pkg/cmd/cli/cmd/expose.go

```diff
@@ -56,7 +56,13 @@ func NewCmdExpose(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.C
 		cmdutil.CheckErr(err)
 		defRun(cmd, args)
 	}
-	cmd.Flags().String("hostname", "", "Set a hostname for the new route")
+	cmd.Flags().String("hostname", "", "Set a hostname for the new route. Optional.")
+	cmd.Flags().String("path", "", "Path that the router watches to route traffic to the service. Optional.")
+	cmd.Flags().String("tls-termination", "", "Indicate a termination type for the generated route. If not set, any termination config will be ignored. Supports 'edge', 'passthrough', or 'reencrypt'.")
```

Comment:
I was talking about both.  If the current behavior is what that message
says, then it seems like the wrong behavior.

On Wed, Nov 25, 2015 at 7:29 AM, Michail Kargakis <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/expose.go
> <https://github.com/openshift/origin/pull/6056#discussion_r45860929>:
>
> > @@ -56,7 +56,13 @@ func NewCmdExpose(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.C
> >  		cmdutil.CheckErr(err)
> >  		defRun(cmd, args)
> >  	}
> > -	cmd.Flags().String("hostname", "", "Set a hostname for the new route")
> > +	cmd.Flags().String("hostname", "", "Set a hostname for the new route. Optional.")
> > +	cmd.Flags().String("path", "", "Path that the router watches to route traffic to the service. Optional.")
> > +	cmd.Flags().String("tls-termination", "", "Indicate a termination type for the generated route. If not set, any termination config will be ignored. Supports 'edge', 'passthrough', or 'reencrypt'.")
>
> @jwforres <https://github.com/jwforres> I just realized you were talking
> about the flag message and not the actual code that implements this. Note
> that I borrowed the description from the swagger spec
> <https://github.com/openshift/origin/blob/003b0435e10e9b37ccbceee0ea93ab3d30a10b07/api/swagger-spec/oapi-v1.json#L18662>
> and now that I'm thinking about it, the code should stay as is if the
> behavior holds true.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6056/files#r45860929>.
>